solr_schema_init/create_isb_core_schema.py

```diff
@@ -216,4 +216,11 @@ def createCollection():
 createField("producedBy_samplingSite_location_latitude", "pfloat", True, True, None)
 createField("producedBy_samplingSite_location_longitude", "pfloat", True, True, None)
 
```

Comment:
New field type and field.  Interestingly `DateRangeField` isn't included as part of the default field types in Solr (though it is part of the distribution).